client/cmdhfmfdes.c

```diff
@@ -913,35 +939,37 @@ static int CmdHF14ADesCreateApp(const char *Cmd) {
     int keylen2 = 1;
     int namelen = 16;
     CLIGetHexWithReturn(1, aid, &aidlength);
+    swap24(aid);
     CLIGetHexWithReturn(2, fid, &fidlength);
+    swap16(fid);
     CLIGetHexWithReturn(3, &keysetting1, &keylen1);
     CLIGetHexWithReturn(4, &keysetting2, &keylen2);
     CLIGetStrWithReturn(5, name, &namelen);
     CLIParserFree();
 
     if (aidlength < 3) {
         PrintAndLogEx(ERR, "AID must have 3 bytes length.");
-        return PM3_EINVARG;
+        return PM3_SNONCES;
     }
 
     if (fidlength < 2) {
         PrintAndLogEx(ERR, "FID must have 2 bytes length.");
-        return PM3_EINVARG;
+        return PM3_SNONCES;
     }
 
     if (keylen1 < 1) {
         PrintAndLogEx(ERR, "Keysetting1 must have 1 byte length.");
-        return PM3_EINVARG;
+        return PM3_SNONCES;
     }
 
     if (keylen1 < 1) {
         PrintAndLogEx(ERR, "Keysetting2 must have 1 byte length.");
-        return PM3_EINVARG;
+        return PM3_SNONCES;
     }
 
     if (namelen > 16) {
         PrintAndLogEx(ERR, "Name has a max. of 16 bytes length.");
-        return PM3_EINVARG;
+        return PM3_SNONCES;
```

Comment:
PM3_SNONCES  is a successfull return code,  used when nonce collecting...  Lets keep the PM3_EINVARG